subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -25,6 +25,7 @@ import static org.gradle.internal.scan.config.BuildScanPluginCompatibility.MIN_S
 class BuildScanPluginSmokeTest extends AbstractSmokeTest {
 
     private static final List<String> GRACEFULLY_UNSUPPORTED_WITHOUT_FAILURE = [
+            "2.0.2",
```

Comment:
Plugin 2.0.2 is still support with Gradle 5.1, right?

Reply:
Yes. The change has been reverted.